accord-core/src/main/java/accord/local/SerializerSupport.java

```diff
@@ -90,11 +93,80 @@ public class SerializerSupport
                 .addAll(APPLY_TYPES)
                 .build();
 
+    private static final Set<MessageType> BOOTSTRAP_TYPES = ImmutableSet.of(MessageType.BOOTSTRAP_ATTEMPT_MARK_BOOTSTRAP_COMPLETE,
+                                                                            MessageType.BOOTSTRAP_ATTEMPT_COMPLETE_MARKER);
+
+    private static Command localOnly(Agent agent, RangesForEpoch rangesForEpoch, Mutable attrs, SaveStatus status, Timestamp executeAt, @Nullable Timestamp executesAtLeast, Ballot promised, Ballot accepted, WaitingOnProvider waitingOnProvider, MessageProvider messageProvider)
```

Comment:
I honestly hate this method, so if anyone can think of a better way im all ears... felt LocalOnly was too special cased in the code that trying to handle in multiple places was error prone

Reply:
Let's leave a `TODO (required)` or `TODO (expected)` to revisit this towards the end of development and clean it up.